entries/t/turhost.com.json

```diff
@@ -0,0 +1,23 @@
+{
+  "Hosting": {
+    "domain": "turhost.com",
+    "url": "https://www.turhost.com/hosting/",
+    "tfa": [
+      "sms",
+      "totp"
+    ],
+    "contact": {
+      "facebook": "turhost",
+      "twitter": "turhost",
+      "linkedin": "turhost",
+      "instagram": "turhost"
+    },
```

Comment:
Once there is a `tfa` section, the `contact` section is not used anymore (which does not support `linkedin` or `instagram` anyway).
```suggestion
```